### PR TITLE
[SPARK-11279] [PYSPARK] Add DataFrame#toDF in PySpark

### PR DESCRIPTION
**Pull request by zjffdu**

### CODE REVIEW

File: python/pyspark/sql/dataframe.py [hunk 1]
```diff
@@ -1266,6 +1266,17 @@ def drop(self, col):
             raise TypeError("col should be a string or a Column")
         return DataFrame(jdf, self.sql_ctx)
 
+    def toDF(self, *cols):
```
Comment (rxin) [inline python/pyspark/sql/dataframe.py]: I think you need to add the ignore utf8 prefix annotation

**Comment by zjffdu** [comment]
Thanks @rxin,  Make another commit to add the annotation

**Comment by rxin** [comment]
Thanks - I've merged this.

**Comment by rxin** [comment]
JIRA doesn't work for me right now. I will close the ticket when it works.

**Pull request closed without merging**